mediator/med_map_mod.F90

```diff
@@ -356,21 +357,27 @@ subroutine med_map_routehandles_initfrom_field(n1, n2, fldsrc, flddst, mapindex,
             srcTermProcessing=srcTermProcessing_Value, &
             ignoreDegenerate=.true., &
             unmappedaction=ESMF_UNMAPPEDACTION_IGNORE, &
+            !DEBUG
```

Comment:
I think you wrote this during our conversation w/ Rocky & Ufuk re: bilinear mapping. Do you want to keep it here?

Reply:
@DeniseWorthen - no that should not be there. Thanks for catching this.